<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolEditorDialog</class>
 <widget class="QDialog" name="ToolEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="2">
      <widget class="QCheckBox" name="ShowPasswordCheckBox">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label7z_3">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="PassEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="PassLabel_5">
       <property name="text">
        <string>Executable</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="BrowsePathButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="PassRetryLabel">
       <property name="text">
        <string>Retry password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="PathEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="TypeComboBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="PassRetryEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="SetSystemAction">
   <property name="text">
    <string>Use system default</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PassEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ToolEditorDialog</receiver>
   <slot>UpdatePassRetryEdits()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PassRetryEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ToolEditorDialog</receiver>
   <slot>UpdatePassRetryEdits()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ToolEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>UpdatePassRetryEdits()</slot>
 </slots>
</ui>
